### DISCUSSION TITLE
What causes there to be no Cache-Control: no-cache header?

### DISCUSSION THREAD

**Discussion by claq2** [Dotnet-Boxed/Templates#142] [Q&A]
In a default WebAPI project, API controller response headers contain Cache-Control: no-cache. When I add an API controller to the MVC 5 Boilerplate template project there is no such response header. What bit of code or config is preventing this?

**Comment by claq2** [comment]
Specifically, my Startup.ConfigureContainer has the following:

    var httpConfig = new HttpConfiguration();
    httpConfig.MapHttpAttributeRoutes();
    httpConfig.Routes.MapHttpRoute(
        name: "DefaultApi",
        routeTemplate: "api/{controller}/{id}",
        defaults: new { id = RouteParameter.Optional }
    );

    IContainer container = CreateContainer();
    app.UseAutofacMiddleware(container);

    // Register MVC Types 
    app.UseAutofacMvc();

    httpConfig.DependencyResolver = new AutofacWebApiDependencyResolver(container);
    app.UseWebApi(httpConfig);

And I'm comparing this setup against the VS 2015 template when selecting an API project.

**Comment by RehanSaeed** [comment]
Web API 2 and MVC 5 are different frameworks. The boilerplate MVC 5 template is not intended to be used for building API's.
